### PR TITLE
Add entering power-off mode

### PR DESCRIPTION
**Pull request by kienvo**
Enter power-off mode on the 2rd press of KEY1

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -87,6 +87,51 @@ void draw_testfb()
...
+		// Disable fb transition while in download mode
+		btn_onOnePress(KEY2, NULL);
+
+		// Take control of the current fb to display 
+		// the Bluetooth animation
+		while (mode == DOWNLOAD) {
+			/* Animation and Bluetooth will be placed here */
+		}
+		// If not being flashed, pressing KEY1 again will 
+		// make the badge goes off:
```
Comment (fcartegnie) [inline src/main.c:123]: fallsthrough, missing break

**Review by fcartegnie — CHANGES_REQUESTED** [review]

File: src/main.c (hunk 1 above)
Comment (kienvo) [reply src/main.c:123]: No need for break as `mode` will be `POWER_OFF` once `change_mode()` get called in button handler. Additionally, if frames are downloaded, the badge will get reset itself.

Comment (fcartegnie) [reply src/main.c:123]: as this is unlabeled, compiler should scream warnings about this

Comment (kienvo) [reply src/main.c:123]: My toolchain (MRS_Toolchain_Linux_x64_V1.91, riscv-none-embed-gcc-8.2.0) was not turn on -Wimplicit-fallthrough by default.

Should I turn on -Wextra and add `__attribute__ ((fallthrough))`?

Comment (fcartegnie) [reply src/main.c:123]: warnings are comment aware.

// fallthrough

**Pull request merged**